indico/modules/events/registration/controllers/api/checkin.py

```diff
@@ -159,3 +160,7 @@ def _process_POST(self):
             'i': [qr_code_version, url, _base64_encode_uuid(self.registration.ticket_uuid)]
         }
         return jsonify(result)
+
+
+class RHCheckinAPIRegistrationPicture(DownloadRegistrationPictureMixin, RH):
```

Comment:
This currently leaks pictures with a not-particularly-secret URL and no authentication requirement.

Ideally we'd ensure that the app includes the bearer token when retrieving this image and then simply inherit from `RHCheckinAPIBase` to get the proper access checks.

Reply:
blob sounds easier tbh